packages/core/src/hooks/useWalletRequest.ts

```diff
@@ -0,0 +1,82 @@
+import { useCallback } from "react";
+import { RpcMessage, RpcTypeToMessageMap } from "starknet-types";
+import { useStarknet } from "~/context/starknet";
+import { UseMutationProps, UseMutationResult, useMutation } from "~/query";
+import { Connector } from "..";
```

Comment:
```suggestion
import { Connector } from "~/connectors/base";
```

Looks better this way :)